Formula/imagemagick.rb

```diff
@@ -126,7 +122,8 @@ def install
     args << "--with-quantum-depth=#{quantum_depth}" if quantum_depth
 
     # versioned stuff in main tree is pointless for us
-    inreplace "configure", "${PACKAGE_NAME}-${PACKAGE_VERSION}", "${PACKAGE_NAME}"
+    inreplace "configure",
```

Comment:
> Apparently I aped @DomT4's practices in #4525 and was told I was wrong.

Eh. Human error. I usually fix them when I find them, but evidently was too focused on fixing the build there to think about it much 😅.

> Aligning the ':' under the '"' was definitely his idea #3827 (comment) :D

@ilovezfs Do you have a book somewhere of everything I've publicly said on style or something? 😄.